src/sif/dynamiccost.cc

```diff
@@ -233,6 +237,17 @@ void DynamicCost::set_use_tracks(float use_tracks) {
                       : (kMinTrackFactor + 2.f * (1.f - use_tracks) * (1.f - kMinTrackFactor));
 }
 
+void DynamicCost::set_use_living_streets(float use_living_streets) {
+  // Calculate factor value based on use preference. Return value
+  // in range [kMaxLivingStreetFactor; 1], if use < 0.5; or
+  // in range [1; kMinLivingStreetFactor], if use > 0.5
```

Comment:
Maybe mention that `living_street_factor` is inversely proportional to `use_living_streets`?

Reply:
done